packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -34,19 +35,19 @@ export interface SetDefaultLoanScheme {
 export interface SetCollateralToken {
   token: number // ----------------| VarUInt{1-9 bytes}, Symbol or id of collateral token
   factor: BigNumber // ------------| 8 bytes unsigned, Collateralization factor
-  priceFeedId: string // ----------| 32 bytes hex string, Txid of oracle feeding the price
+  priceFeedId: CurrencyPair // ----| priceFeedId: CurrencyPair // ----| c1 = VarUInt{1-9 bytes} + c1 bytes UTF encoded string for token + c2 = VarUInt{1-9 bytes} + c2 bytes UTF encoded string for currency, token/currency pair to use for price of token
   activateAfterBlock: number // ---| 4 bytes unsigned, Changes will be active after the block height
 }
 
 /**
  * SetLoanToken DeFi Transaction
  */
 export interface SetLoanToken {
-  symbol: string // ------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string, Symbol or id of collateral token
-  name: string // --------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string, Token's name, no longer than 128 characters
-  priceFeedId: string // -------| 32 bytes, hex string, Txid of oracle feeding the price
-  mintable: boolean // ---------| 1 byte, mintable, Token's 'Mintable' property
-  interest: BigNumber // -------| 8 bytes unsigned, interest rate
+  symbol: string // -------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string, Symbol or id of collateral token
+  name: string // ---------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string, Token's name, no longer than 128 characters
+  priceFeedId: CurrencyPair // --| priceFeedId: CurrencyPair // ----| c1 = VarUInt{1-9 bytes} + c1 bytes UTF encoded string for token + c2 = VarUInt{1-9 bytes} + c2 bytes UTF encoded string for currency, token/currency pair to use for price of token
```

Comment:
```suggestion
  currentPair: CurrencyPair // --| priceFeedId: CurrencyPair // ----| c1 = VarUInt{1-9 bytes} + c1 bytes UTF encoded string for token + c2 = VarUInt{1-9 bytes} + c2 bytes UTF encoded string for currency, token/currency pair to use for price of token
```

should be renamed here and on c++ since to prevent confusion

Reply:
Ok I shall fix this in the next PR

Reply:
@surangap

Reply:
@Fuxing currencyPair right? Not currentPair

Reply:
@jingyi2811 needs to communicate with cpp side also.

Reply:
I have logged a ticket in ain. https://github.com/DeFiCh/ain/issues/754